lib/js/styles/ItemsTypography.vue

```diff
@@ -3,7 +3,7 @@
 		<div v-for="item in items" :key="item.id" class="itemsListItem">
 			<div v-if="item.label" class="itemsListItem__label">{{ item.label }}</div>
 			<div v-if="item.value" class="itemsListItem__value">{{ item.value }}</div>
-			<div v-if="item.ratio" class="itemsListItem__value">{{ item.ratio }}</div>
+			<div v-if="item.ratio || item.value" class="itemsListItem__value">{{ item.ratio }}</div>
```

Comment:
do we want to display empty div, when `ratio` is empty and `value` is set?

Reply:
Yes, because without this we have that case:
![Screenshot 2022-12-01 at 14 17 36](https://user-images.githubusercontent.com/4208462/205062719-aaf28563-6864-4251-8d01-dbaf7c365ce6.png)

Reply:
https://github.com/bethinkpl/design-system/pull/235#discussion_r1037107320

Maybe `<table>` solves your problems?

Reply:
I converted.